src/Converters.ts

```diff
@@ -83,13 +86,13 @@ export function getBindingDefinitions(info: FunctionInfo): IBindingDefinition[]
     .map(name => { return { 
         name: name,
         type: bindings[name].type || "", 
-        direction: getDirectionName(bindings[name].direction) || ""
+        direction: <BindingDirection> getDirectionName(bindings[name].direction)
```

Comment:
Type assertions are dangerous. Consider creating an "isBindingDirection" helper method that does type guarding for you, rather than asserting the type. https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-definied-type-guards